### ISSUE TITLE
Google JS Code is not inserted

### ISSUE THREAD

**Issue by ghost** [backdrop-contrib/googleanalytics#24]
I installed it.

I created the old-style (non-GA4) property.

I plugged in the UA-code.

But the module doesn't insert anything into the footer.

https://dev.edengardenclub.com/

**Comment by jenlampton** [comment]
You may not see anything in the footer of your site. You can tell if the code has been added to the page by searching for your UA-code in the source code. You should find it in a `<script>` tag in the `<head>`. 

If the code really is missing from the page, I'd recommend switching back to the Basis theme, and checking again. It's possible that the theme you are using has moved or removed some code that's necessary.

We have recently updated the module, so if you would like to switch to GA4, that's now also an option. I'd encourage you to try again.

**Comment by sudipto68** [comment]
I have installed Google Analytics on my backdrop site and configured the GA4 account and copied the Web property ID into my site's GA configuration.
But only one line is added to the site header including the G-ID(web property id)
The line is something like this:
`<script>(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)`

But I have found a script on the GA account under the "View Tag instruction" Menu that needs to be added to the site header to collect data. That script is something like this:
```
<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-MYID"></script>
```

```
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-MYID');
</script>
```

But the last script is not added to the site header. Do I need to make any other changes to add this to the header part?
